frontend/src/pages/LearningCourse/ActionButton/styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import { Button } from '@mui/material';
+
+import theme from 'themeSettings';
+
+export const ActionButtonWrapper = styled('div')({
+  display: 'inline-block',
+  verticalAlign: 'top',
+  width: '155px',
+  marginLeft: '40px',
+  color: 'white',
```

Comment:
this color is presented in the theme palette
you can use theme.palette.secondary.main or Hex color code